src/components/Header/Header.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { ContentWrapper } from '../../common';
+import { HeaderWrapper, Logo } from './HeaderElements';
+
+export const Header = () => (
+  <HeaderWrapper>
+   <ContentWrapper>
+    <Logo src='./logo.png' alt="Logo" />
```

Comment:
It's a better practice do it like following:

```js
import logoImg from './logo.png';

<Logo src={logoImg} alt="Logo" />
```

You can even do this in the Logo component itself.

Reply:
I tried this way and it's not working, so I googled another way.

![image](https://user-images.githubusercontent.com/51786805/83325407-4a017a80-a26c-11ea-8475-61d0e0db46fd.png)